### DISCUSSION TITLE
crates.io version checker

### DISCUSSION THREAD

**Discussion by gruberb** [zed-industries/zed#20238] [Feature Requests]
I am curious if this is a) planned or b) possible to write an extension for (I assume it is not (yet?)).

When I work with a Rust project, and add a dependency in the `Cargo.toml` file, I would like to get a dropdown or an auto-complete of the latest version and possible feature flags available for this crate. 

Another IDE has the "version dropdown" and it's really handy. I would like to bring this over to Zed, and wonder if someone could tell me if that's planned or possible to write an extension for?

**Comment by ActuallyHappening** [comment]
I would also find this useful as an optional extension, like what VS Code currently has here: https://marketplace.visualstudio.com/items?itemName=BarbossHack.crates-io

**Comment by aurexav** [comment]
I really need this.

<img width="712" alt="image" src="https://github.com/user-attachments/assets/64402ddd-3dfe-44ce-801f-3e7bdde85a74" />

**Comment by dxvid-pts** [comment]
Related: https://github.com/zed-industries/zed/discussions/32053

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
